deploy/helm/kubecf/templates/brain-tests.yaml

```diff
@@ -0,0 +1,223 @@
+# psp-role       | predefined ? -> scf default account
+# configgin-role | predefined ? -> scf default account
+
+# Service account, roles, and cluster role for the brain tests.
+# Raw kube objects going into the chart.
+# TODO: Make this conditional.
+
+# Service account "tests-brain" is used by the instance group "acceptance-tests-brain"
+---
+apiVersion: "v1"
+kind: "ServiceAccount"
+metadata:
+  name: "{{ .Values.deployment_name }}-tests-brain"
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/component: "{{ .Values.deployment_name }}-tests-brain"
```

Comment:
The `component` being templated seems odd.

Reply:
I followed what I saw with the `eirini` template.